docs/.eleventy.js

```diff
@@ -55,11 +56,23 @@ module.exports = function(eleventyConfig) {
     // Load site-specific data
     const siteName = process.env.ESLINT_SITE_NAME || "en";
 
+    /**
+     * Determines whether we are in the prerelease phase.
+     * @returns {Promise<boolean>} `true` if there is a prerelease of the next major version, `false` otherwise.
+     */
+    async function isPrereleasePhase() {
+        const eslintVersions = await require("./src/_data/eslintVersions")();
+
+        return eslintVersions.items.some(item => item.branch === "next");
+    }
```

Comment:
I just noticed that this would be an extra fetch on Netlify.

What do you think about the following, instead of adding a global variable `isPrereleasePhase`:

1. In `eslintVersions.js` we set `data.isPrereleasePhase`.
2. In `docs.html` we use `eslintVersions.isPrereleasePhase`.

Reply:
Done!